tests/rados/rados_prep.py

```diff
@@ -0,0 +1,327 @@
+    cmd = "ceph config dump | grep 'mgr/alerts'"
+    log.info(node.shell([cmd]))
+
+    # Disabling and enabling the email alert module after setting all the config
+    try:
+        cmd = "ceph mgr module disable alerts"
+        node.shell([cmd])
+        time.sleep(2)
+        cmd = "ceph mgr module enable alerts"
+        node.shell([cmd])
+        time.sleep(2)
+    except Exception as err:
+        log.error("Error while enabling/disabling alerts module after configuration")
+        log.error(err)
+        return False
+
+    # Triggering email alert
+    try:
+        cmd = "ceph alerts send"
+        node.shell([cmd])
+    except Exception as err:
+        log.error("Error while Sending email alerts")
+        log.error(err)
+        return False
+
+    log.info("Email alerts configured on the cluster")
+    return True
+
+
+def create_erasure_pool(node: CephAdmin, name: str, **kwargs) -> bool:
+    """
+    Creates a erasure code profile and then creates a pool with the same
+    Args:
+        node: Cephadm node where the commands need to be executed
+        name: Name of the profile to create
+        **kwargs: Any other param that needs to be set in the EC profile
+    Returns: True -> pass, False -> fail
+
+    """
+    failure_domain = kwargs.get("crush-failure-domain", "osd")
+    k = kwargs.get("k", 3)
+    m = kwargs.get("m", 2)
+    plugin = kwargs.get("plugin", "jerasure")
+    pool_name = kwargs.get("pool_name")
+    profile_name = f"ecprofile_{name}"
+
+    # Creating a erasure coded profile with the options provided
+    cmd = (
+        f"ceph osd erasure-code-profile set {profile_name}"
+        f" crush-failure-domain={failure_domain} k={k} m={m} plugin={plugin}"
+    )
+    try:
+        node.shell([cmd])
+    except Exception as err:
+        log.error(f"Failed to create ec profile : {profile_name}")
+        log.error(err)
+        return False
+
+    cmd = f"ceph osd erasure-code-profile get {profile_name}"
+    log.info(node.shell([cmd]))
+    # Creating the pool with the profile created
+    if not create_pool(
+        node=node,
+        ec_profile_name=profile_name,
+        disable_pg_autoscale=True,
+        **kwargs,
+    ):
+        log.error(f"Failed to create Pool {pool_name}")
+        return False
+    log.info(f"Created the ec profile : {profile_name} and pool : {pool_name}")
+    return True
+
+
+def create_pool(node: CephAdmin, pool_name: str, pg_num: int = 64, **kwargs) -> bool:
+    """
+    Create a pool named from the pool_name parameter.
+     Args:
+        node: Cephadm node where the commands need to be executed
+        pool_name: name of the pool being created.
+        pg_num: initial number of pgs.
+        kwargs: Any other args that need to be passed
+
+     Returns: True -> pass, False -> fail
+    """
+
+    log.info(f"creating pool_name {pool_name}")
+    cmd = f"ceph osd pool create {pool_name} {pg_num} {pg_num}"
+    if kwargs.get("ec_profile_name"):
```

Comment:
if we try to have a 1:1 mapping between the key and cli, it would be easier to loop and be close to the CLI options.